minio/helm/minio/values.yaml

```diff
@@ -5,112 +5,20 @@ secret:
 storageClass: ""
 
 minio:
-  ## Provide a name in place of minio for `app:` labels
```

Comment:
I'm assume the red diffs on this file is just removing duplication of existing defaults?

Reply:
Yeah, I did a major upgrade of the MinIO chart and removed all the default values since that causes a lot more maintenance overhead.